binchihua/src/ini/config.c

```diff
@@ -126,3 +127,136 @@ void free_topics_hashmap(struct hashmap **topics) {
         }
         hashmap_free(*topics);
 }
+
+
+bool is_IP(char *str) {
```

Comment:
I think this should go into parse-util.c
Also, systemd has quite some IP addr utils in in-addr-util.c

Reply:
Please review again.
I think that systemd IP utils is overkill for what I need